sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -507,6 +507,82 @@ describe("ManagedIdentityCredential", function () {
     }
   });
 
+  it("sends an authorization request correctly in an App Service 2019 environment by client id", async () => {
```

Comment:
Note: Follow up with the crew in standup - do we need to remove support for App service 2017 version?

Reply:
ok turns out we should not remove 2017! So this PR is good 🙂